src/observer.js

```diff
@@ -348,20 +344,13 @@ function mixinLifecycleEvents(target) {
     }
 }
 
-export const Observer = observer(({ children, inject: observerInject, render }) => {
+// TODO: use mobx-react-lite version?
```

Comment:
Should be safe enough imo since it is used in the component tree, but let's add a test to see what happens when wrapped component is class one.

Reply:
done